app/src/main/java/org/jellyfin/androidtv/details/MovieDetailsFragment.kt

```diff
@@ -49,12 +48,17 @@ class MovieDetailsFragment(item: Movie) : BaseDetailsFragment<Movie>(item) {
 		(detailsRow.actionsAdapter as ActionAdapter).apply {
 			reset()
 
-			if (item.canResume) add(ResumeAction(context!!, item).apply { icon = context!!.getDrawable(R.drawable.ic_resume) })
-
-			add(PlayFromBeginningAction(context!!, item).apply { icon = context!!.getDrawable(R.drawable.ic_play) })
-			add(ToggleWatchedAction(context!!, item).apply { icon = context!!.getDrawable(R.drawable.ic_watch) })
-			add(ToggleFavoriteAction(context!!, item).apply { icon = context!!.getDrawable(R.drawable.ic_heart) })
-			add(Action(0, "More").apply { icon = context!!.getDrawable(R.drawable.ic_more) }) // Show menu with more options
+			if (item.canResume) add(ResumeAction(context!!, item))
+			add(PlayFromBeginningAction(context!!, item))
+			// TODO: Bring this back once we have a more understandable queue implementation for users
+			//add(ToggleWatchedAction(context!!, item))
```

Comment:
I think you meant comment `AddToQueueAction`

Reply:
Uh, yes